web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/controller/InstitutionController.java

```diff
@@ -60,22 +60,22 @@ public ApiKeysResource getInstitutionApiKeys(
     @PostMapping("/{institutionId}/api-keys")
     @ResponseStatus(HttpStatus.CREATED)
     @ApiOperation(value = "", notes = "${swagger.api.institution.createInstitutionApiKeys}")
-    public ApiKeysResource createInstitutionApiKeys(@ApiParam("${swagger.model.institution.id}")
-                                                    @PathVariable("institutionId") String institutionId) {
+    public List<ApiKeysResource> createInstitutionApiKeys(@ApiParam("${swagger.model.institution.id}")
+                                                          @PathVariable("institutionId") String institutionId) {
         log.trace("createInstitutionApiKeys start");
         log.debug("createInstitutionApiKeys institutionId = {}", institutionId);
-        InstitutionApiKeys institutionKeys = apiManagementService.createInstitutionKeys(institutionId);
-        ApiKeysResource apiKeysResource = ApiManagerMapper.toApiKeysResource(institutionKeys);
-        log.debug(LogUtils.CONFIDENTIAL_MARKER, "createInstitutionApiKeys result = {}", apiKeysResource);
+        List<InstitutionApiKeys> institutionKeys = apiManagementService.createInstitutionKeysList(institutionId);
+        List<ApiKeysResource> apiKeysResourceList = ApiManagerMapper.toApikeysResourceList(institutionKeys);
+        log.debug(LogUtils.CONFIDENTIAL_MARKER, "createInstitutionApiKeys result = {}", apiKeysResourceList);
         log.trace("createInstitutionApiKeys end");
-        return apiKeysResource;
+        return apiKeysResourceList;
     }
     
     @PostMapping("/{institutionId}/api-keys/primary/regenerate")
     @ResponseStatus(HttpStatus.NO_CONTENT)
     @ApiOperation(value = "", notes = "${swagger.api.institution.regeneratePrimaryKey}")
-    public void regeneratePrimaryKey(@ApiParam("${swagger.model.institution.id}")
-                                     @PathVariable("institutionId")String institutionId){
+    public void regeneratePrimaryKey(@ApiParam("${swagger.model.institution.subscription.id}")
+                                     @PathVariable("institutionId") String institutionId) {
```

Comment:
swagger description changed to subscriptionId but path variable stayed institutionId why?

Reply:
fixed